### PR TITLE
Docs updated for notebook dynamic role changes

### PR DESCRIPTION
**Pull request by dustins**
*Issue #, if available:*

*Description of changes:*]
- added tabs for example Notebook roles with and without dynamic roles enabled
- fixed text about dynamic roles
  - misc grammar
  - replaced `MLSpace` with `{{ $params.APPLICATION_NAME }}`

Screenshot 2024-06-13 at 22 08 59

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/docs/admin-guide/getting-started.md [hunk 1]
```diff
@@ -265,6 +265,6 @@ Listed below are security best practices and design principles implemented by th
...
...the execution role associated with the instance. The creation of AWS Identity & Access Management (IAM) roles within an account is typically managed by a separate organizational process that requires manual review. Since the {{ $params.APPLICATION_NAME }} application’s goal is to provide users with a frictionless mechanism to self-organize and create projects for collaboration, the turnaround time for role creation is a limiting factor. The use of coarse-grained policies to reduce the number of roles required to manage the {{ $params.APPLICATION_NAME }} environment limits the ability to isolate project resource access to only the authorized {{ $params.APPLICATION_NAME }} users when requests originate from a SageMaker notebook instance. IdP credentials cannot be stored directly on the notebook instances for a number of reasons, including security concerns. The “identity” of the notebook becomes obfuscated if a common IAM role is implemented across the user community. Additionally, users require integration with external systems to access data, code repositories, and software libraries relevant to their ML use cases, which further increases the need for more granular access controls.
```
Comment (LightSeekerSC) [inline frontend/docs/admin-guide/getting-started.md:268]: Hard to tell what changed in this line and the one below via github. Can you clarify what changed?

File: frontend/docs/admin-guide/install.md [hunk 2]
```diff
@@ -95,6 +95,250 @@ In order to create the default {{ $params.APPLICATION_NAME }} notebook policy an
 1. Login to your AWS account and go to the Policies section of the IAM Service in the AWS Console
 2. Create a new policy with using the JSON editor and paste the following in the text area (after replacing the placeholder variables):
 
+:::tabs
+== Dynamic Roles Enabled
```
Comment (LightSeekerSC) [inline frontend/docs/admin-guide/install.md:99]: Was wondering what the tabs were for. Very nice! Love this addition.

**Review by LightSeekerSC — COMMENTED** [review]
Mostly looks good to me, just curious what we changed in the getting started blurbs because github isn't rendering the word removals and additions clearly.

File: frontend/docs/admin-guide/getting-started.md (hunk 1 above)
Comment (dustins) [reply frontend/docs/admin-guide/getting-started.md:268]: Yeah I am not sure why it isn't showing the changes…

This line…
- `provide users a` -> `provide users with a`
- `relevant to their ML use cases which further increases` -> `relevant to their ML use cases, which further increases`

One below…
- `attaches a standardized set of IAM policies` -> `attach a standardized set of IAM policies`
- a couple `MLSpace` -> `{{ $params.APPLICATION_NAME }}`

File: frontend/docs/.vitepress/theme/custom.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:root:not(.dark) .plugin-tabs--content div[class*="language-"] {
+    background-color: inherit !important;
```
Comment (AlejandroRigau) [inline frontend/docs/.vitepress/theme/custom.scss:2]: Why did this need to get added? Dont our docs already have some theme?

**Review by AlejandroRigau — APPROVED** [review]
I left a comment but I dont think it will be an issue. Approving. I like the tabs too

**Review by estohlmann — APPROVED** [review]

File: frontend/docs/.vitepress/theme/custom.scss (hunk 3 above)
Comment (dustins) [reply frontend/docs/.vitepress/theme/custom.scss:2]: If you see the `.plugin-tabs--content` part of the selector that style was overriding the background color in code blocks. I wanted to make code in tabs match code outside of tabs.

**Pull request merged**